### PR TITLE
Move test procedures and operations that related to searchable snapshots to seprate files in nyc_taxis workload

### PR DESCRIPTION
**Pull request by tlfeng**
### Description
The PR moves the new "test procedures" and "operations" that added in commit https://github.com/opensearch-project/opensearch-benchmark-workloads/commit/ed7b2b6690ac03d5822c9d0f835d7ab90faa7b44 which related to Searchable Snapshots feature to separate files.

With the change, the 2 files [nyc_taxis/operations/default.json](https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/1/nyc_taxis/test_procedures/default.json) and [nyc_taxis/operations/default.json](https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/1/nyc_taxis/operations/default.json) will be restored to the previous state before the above commit. It will help moving the contents that related to Searchable Snapshots feature to a separate workload.

Testing:
```
opensearch-benchmark execute_test \
--workload-path=/home/ftianli/github/opensearch-benchmark-workloads/nyc_taxis --test-procedure append-no-conflicts --test-mode

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

[INFO] Preparing for test execution ...
[INFO] Executing test with workload [nyc_taxis], test_procedure [append-no-conflicts] and provision_config_instance ['defaults'] with version [3.0.0-SNAPSHOT].

Running delete-index                                                           [100% done]
Running create-index                                                           [100% done]
Running check-cluster-health                                                   [100% done]
Running index                                                                  [100% done]
Running refresh-after-index                                                    [100% done]
Running force-merge                                                            [100% done]
Running refresh-after-force-merge                                              [100% done]
Running wait-until-merges-finish                                               [100% done]
Running default                                                                [100% done]
Running range                                                                  [100% done]
Running distance_amount_agg                                                    [100% done]
Running autohisto_agg                                                          [100% done]
Running date_histogram_agg                                                     [100% done]

```

```
opensearch-benchmark execute_test \
--workload-path=/home/ftianli/github/opensearch-benchmark-workloads/nyc_taxis --test-procedure searchable-snapshot \
--workload-params="s3_bucket_name:opensearch-snapshot-0405,s3_bucket_region:us-west-2" 
--provision-config-instance-params="/home/ftianli/Downloads/benchmark/searchable_snapshot/provision-config-instance-params.json" \
--opensearch-plugins="repository-s3" --plugin-params="/home/ftianli/Downloads/benchmark/searchable_snapshot/plugin-params.json" --test-mode

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

[INFO] Preparing for test execution ...
[INFO] Executing test with workload [nyc_taxis], test_procedure [searchable-snapshot] and provision_config_instance ['defaults'] with version [3.0.0-SNAPSHOT].

Running delete-index                                                           [100% done]
Running create-index                                                           [100% done]
Running check-cluster-health                                                   [100% done]
Running index                                                                  [100% done]
Running refresh-after-index                                                    [100% done]
Running force-merge                                                            [100% done]
Running refresh-after-force-merge                                              [100% done]
Running wait-until-merges-finish                                               [100% done]
Running create-snapshot-repository                                             [100% done]
Running delete-snapshot                                                        [100% done]
Running create-snapshot                                                        [100% done]
Running wait-for-snapshot-creation                                             [100% done]
Running delete-local-index                                                     [100% done]
Running restore-snapshot                                                       [100% done]
Running default                                                                [100% done]
Running range                                                                  [100% done]
Running distance_amount_agg                                                    [100% done]
Running autohisto_agg                                                          [100% done]
Running date_histogram_agg                                                     [100% done]
```

### Issues Resolved
n/a

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by gkamat** [comment]
Taking a look.

**Comment by tlfeng** [comment]
@gkamat Thank you! I haven't tested the change yet, and I will reach out to you when I have tested.

**Comment by tlfeng** [comment]
@gkamat @IanHoang I have manual tested the code change by running both default (append-no-conflicts) and searchable-snapshot test procedures. Please let me know if you have any opinions. This change is a bit trivial, but helps when moving the "searchable-snapshot" out of nyc_taxis workload in the future.

**Review by IanHoang — APPROVED** [review]
Reviewed the changes and checks out to me. Thanks for reworking this and moving it to separate files!

File: nyc_taxis/operations/default.json [hunk 1]
```diff
@@ -25,54 +25,6 @@
       "retry-until-success": true,
       "include-in-reporting": false
     },
-    {
```
Comment (IanHoang) [inline nyc_taxis/operations/default.json:28]: Looks good. It's better that this is moved to a separate file. We'll create a statement in this workload that states that `default.json` in operations and test_procedures shouldn't be updated directly but rather in separate files (unless to fix breaking changes or other exceptions)

**Comment by gkamat** [comment]
Noticed a typo in the file name: `nyc_taxis/test_procedures/seachable-snapshot.json`.  Could you check in a change to correct this, @tlfeng?  Thanks.

**Pull request merged**